installer/templates/new/widgets/text/text.scss

```diff
@@ -19,14 +19,20 @@ $moreinfo-color:    rgba(255, 255, 255, 0.7);
 
   .more-info {
     color: $moreinfo-color;
+    font-size: 24px;
   }
 
   .updated-at {
     color: rgba(255, 255, 255, 0.7);
   }
 
 
-  &.large h3 {
+  & h3 {
     font-size: 65px;
   }
 }
+
+.widget-text-custom {
+
+    background-color: #ff0000;
+}
```

Comment:
I think that user defined styles belong elsewhere. If only a few rules are to be added application.scss seems like a better fit, otherwise they can be placed in a separate file imported in application.scss.